Test.Microsoft.NET.Build.Containers.Filesystem/Evaluator.cs

```diff
@@ -0,0 +1,98 @@
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Locator;
+
+namespace Test.Microsoft.NET.Build.Containers;
+
+public static class Evaluator {
```

Comment:
Not sure I understand this class name

Reply:
Renamed it to hopefully be less obscure.